Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -735,4 +736,18 @@ protected boolean isConditionsDefaultValueEnabled() {
         }
         return Boolean.TRUE;
     }
+
+    public static String appendPrefixHomeCommunityID(final String homeCommunityId) {
+        return checkPrefixBeforeAppend(homeCommunityId, "urn:oid:");
+    }
+
+    public static String checkPrefixBeforeAppend(final String checkValue, final String checkPrefix) {
+        final String tempValue = checkValue.toLowerCase();
+        final String tempPrefix = checkPrefix.toLowerCase();
+        if(tempValue.indexOf(tempPrefix) == -1){
```

Comment:
use startsWith instead of indexOf. indexOf will return position even if the communityId is defined as 67i8urn:oid